test/route-data.js

```diff
@@ -1,93 +0,0 @@
-"use strict";
```

Comment:
I didn't think this file was necessary.

Reply:
right, the `.js` files in `src` and `test` can be removed